scripts/startup-local.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+echo "Starting hardhat..."
+yarn hardhat node &
+
+echo "Waiting for hardhat to be ready..."
+until $(curl --output /dev/null --silent --head --fail http://localhost:8545); do
+    echo "."
+    sleep 0.25
+done
+echo "Hardhat is now ready!"
+
+echo "Deploying contracts (logs in /var/log/deploy.log)"
+yarn hardhat run scripts/deploy-local.js --network localhost > /var/log/deploy.log
+if ! grep '==== config data ====' /var/log/deploy.log >/dev/null ; then
+  echo "Deploy failed (config data not found in deploy logs). Exiting."
+  exit 1
+fi
+
+echo "Creating config file ($1):"
+grep -A100 '==== config data ====' /var/log/deploy.log | tail -n +2 > $1
+
+echo "Creating healthcheck flag /var/log/done.flag"
+touch /var/log/done.flag
```

Comment:
where is this flag being used?

Reply:
See: https://github.com/rnd-inc/toucan-bridge-service/pull/80#discussion_r913920662